openml/testing.py

```diff
@@ -104,9 +112,39 @@ def tearDown(self):
         openml.config.server = self.production_server
         openml.config.connection_n_retries = self.connection_n_retries
 
+    @classmethod
```

Comment:
I'm wondering whether you need to add some synchronization as the tests are executed in parallel?

Reply:
This function just adds to a dictionary and even concurrent executions will `append` a different ID, will it not? Are there any obvious edge case that can break this design? Also, does the parallel distribution (on Travis) create multiple copies of a static variable shared by the class definition?